### PR TITLE
Added `power_wait_timer_friendly` to inform users about AC power time limit

### PR DESCRIPTION
**Pull request by dan-snelson**
Greetings, @grahampugh!

This PR is _nearly_ complete and could use additional testing. (Also, please pardon my attempt at machine translation.)

We wanted a way to inform users of the amount of time they had to connect AC power. We also wanted to inform them when they failed to connect AC in the specified time period.

**Sample Command:**
`bash ./erase-install.sh --reinstall --current-user --depnotify --check-power --power-wait-limit=11`

**Sample Dialog Boxes:**
`jamfHelper`
Screen Shot 2021-10-30 at 6 35 54 PM
Screen Shot 2021-10-30 at 6 36 05 PM

`osascript`
Screen Shot 2021-10-30 at 6 36 34 PM
Screen Shot 2021-10-30 at 6 36 45 PM

### CODE REVIEW

**Comment by grahampugh** [comment]
I'm intrigued by your comment:

> This PR is nearly complete

What else did you have in mind?

**Comment by dan-snelson** [comment]
> … additional testing. (Also, please pardon my attempt at machine translation.)

File: erase-install.sh [hunk 1]
```diff
@@ -180,10 +180,15 @@ dialog_power_title_de="Warten auf AC-Netzteil"
 dialog_power_title_nl="Wachten op Stroomadapter"
 dialog_power_title_fr="En attente de l'alimentation secteur"
 
-dialog_power_desc_en="Please connect your computer to power using an AC power adapter. This process will continue once AC power is detected."
-dialog_power_desc_de="Bitte schließen Sie Ihren Computer mit einem AC-Netzteil an das Stromnetz an. Dieser Prozess wird fortgesetzt, sobald die AC-Stromversorgung erkannt wird."
-dialog_power_desc_nl="Sluit uw computer aan met de stroomadapter. Zodra deze is gedetecteerd gaat het proces verder"
-dialog_power_desc_fr="Veuillez connecter votre ordinateur à un adaptateur secteur. Ce processus se poursuivra une fois que l'alimentation secteur sera détectée."
+dialog_power_desc_en="Please connect your computer to power using an AC power adapter. This process will continue if AC power is detected within the next:"
+dialog_power_desc_de="Bitte schließen Sie Ihren Computer mit einem AC-Netzteil an das Stromnetz an. Dieser Prozess wird fortgesetzt, sobald die AC-Stromversorgung erkannt wird innerhalb der nächsten:"
```
Comment (grahampugh) [inline erase-install.sh:184]: The sentence is a bit tricky to do grammatically in German as the second verb should be at the end (after the time). Let's try:

```
Bitte schließen Sie Ihren Computer mit einem AC-Netzteil an das Stromnetz an. Dieser Prozess wird fortgesetzt, sobald die AC-Stromversorgung innerhalb der folgende Zeitdauer erkannt wird:
```

Comment (grahampugh) [reply erase-install.sh:184]: (I don't mind making these changes after the merge)

Comment (dan-snelson) [reply erase-install.sh:184]: Thanks, Obi-@grahampugh!

**Comment by dan-snelson** [comment]
Yippee!

**Comment by dan-snelson** [comment]
@grahampugh Looks like the following block didn't make it into `25.0-rc`:

https://github.com/dan-snelson/erase-install/commit/506e5610676287b0aace190ecc083f9537587e92#diff-d2e5235ccd355bc15e5860aa8901fd7502d707d3e1082011769156e677f8a6f9L1245-R1260

**Comment by grahampugh** [comment]
Please can you make a new PR with the missing block? I don't really understand how it could be missing, but possibly because I switched the base from `main` to `v25.0-rc`?

**Pull request merged**